pkg/scheduler/cache/cache.go

```diff
@@ -941,6 +941,11 @@ func (sc *SchedulerCache) Client() kubernetes.Interface {
 	return sc.kubeClient
 }
 
+// Client returns the volcano clientSet
```

Comment:
VCClient maybe better
```suggestion
// VCClient returns the volcano clientSet
```